### PR TITLE
Add localization for UI strings in Korean

### PR DESCRIPTION
**Pull request by stepankuzmin**
Hey 👋

This PR adds translations for UI strings we use in GL JS into Korean. We translated the UI strings automatically using the Google Translate API, and we need to verify that the translation is correct.

To help us verify the translations, you can open the `Files changed` tab on this page and compare the translations with the original file [`src/ui/default_locale.js`](https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js).

If the translation looks good, you can approve it using the `Review changes` button in the `Files changed` tab and selecting the `Approve` radio button when submitting the review.

To improve the machine translation, you can click the plus sign next to the line you want to enhance and select the suggestion button on the toolbar.

After making the suggestions, you can mark the review as finished using the `Review changes` button and selecting the `Request changes` radio button.

Thanks for helping 🙌

### CODE REVIEW

File: src/ui/locales/ko.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "FullscreenControl.Exit": "전체화면 종료",
+    "GeolocateControl.FindMyLocation": "내 위치 찾기",
+    "GeolocateControl.LocationNotAvailable": "위치를 사용할 수 없음",
+    "LogoControl.Title": "맵박스 로고",
+    "Map.Title": "지도",
+    "NavigationControl.ResetBearing": "방위를 북쪽으로 재설정",
+    "NavigationControl.ZoomIn": "확대",
+    "NavigationControl.ZoomOut": "축소",
+    "ScaleControl.Feet": "피트",
+    "ScaleControl.Meters": "중",
```
Comment (cs09g) [inline src/ui/locales/ko.js:16]:
```suggestion
    "ScaleControl.Meters": "미터",
```

File: src/ui/locales/ko.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "ScaleControl.Kilometers": "km",
```
Comment (cs09g) [inline src/ui/locales/ko.js:17]:
```suggestion
    "ScaleControl.Kilometers": "킬로미터",
```

File: src/ui/locales/ko.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "ScaleControl.Miles": "나",
```
Comment (cs09g) [inline src/ui/locales/ko.js:18]:
```suggestion
    "ScaleControl.Miles": "마일",
```

File: src/ui/locales/ko.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "ScaleControl.NauticalMiles": "nm",
```
Comment (cs09g) [inline src/ui/locales/ko.js:19]:
```suggestion
    "ScaleControl.NauticalMiles": "해리",
```

File: src/ui/locales/ko.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    "ScrollZoomBlocker.CtrlMessage": "ctrl + 스크롤을 사용하여 지도 확대/축소",
+    "ScrollZoomBlocker.CmdMessage": "⌘ + 스크롤을 사용하여 지도를 확대/축소하세요.",
```
Comment (cs09g) [inline src/ui/locales/ko.js:21]:
```suggestion
    "ScrollZoomBlocker.CmdMessage": "⌘ + 스크롤을 사용하여 지도 확대/축소",
```

File: src/ui/locales/ko.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+const locale = {
+    "AttributionControl.ToggleAttribution": "속성 전환",
+    "AttributionControl.MapFeedback": "지도 피드백",
+    "FullscreenControl.Enter": "전체 화면으로 전환",
+    "FullscreenControl.Exit": "전체화면 종료",
+    "GeolocateControl.FindMyLocation": "내 위치 찾기",
+    "GeolocateControl.LocationNotAvailable": "위치를 사용할 수 없음",
+    "LogoControl.Title": "맵박스 로고",
+    "Map.Title": "지도",
+    "NavigationControl.ResetBearing": "방위를 북쪽으로 재설정",
```
Comment (yseo93) [inline src/ui/locales/ko.js:12]: The Korean above translates  => reset bearing to North. The literal translation should be "방위를 초기화". I am not sure if "North" indicates "Navigation reset". If that's the case, above Korean translation is fine.

Comment (cs09g) [reply src/ui/locales/ko.js:12]: Check [this](https://github.com/mapbox/mapbox-gl-js/pull/11733) link.
> English: Reset bearing to north

Comment (yseo93) [reply src/ui/locales/ko.js:12]: Ah. Thanks! Makes sense.

**Pull request merged**